code/modules/clothing/spacesuits/rig/suits/light.dm

```diff
@@ -92,7 +92,8 @@
 		/obj/item/rig_module/voice,
 		/obj/item/rig_module/ai_container,
 		/obj/item/rig_module/datajack,
-		/obj/item/rig_module/self_destruct
+		/obj/item/rig_module/self_destruct,
+		/obj/item/rig_module/actuators
```

Comment:
That looks like you are merging something from dev into master ?

Reply:
I have no idea. I'm 99% sure that came with what Skull did.

Reply:
Wait. Maybe that's because Doc's PR is in here for some reason too. God damnit.